PyExpLabSys/drivers/scpi.py

```diff
@@ -81,6 +84,11 @@ def scpi_comm(self, command, expect_return=False):
             else:
                 self.comm_dev.write(command_text)
                 return_string = 'command_text'
+
+        if self.interface == 'gpib':
+            self.comm_dev.write(command_text)
+            if (command.find('?') > -1) or expect_return:
```

Comment:
I find `"?" in command` a little easier to parse

Reply:
Agreed.